router/config.go

```diff
@@ -101,7 +103,7 @@ type Config struct {
 		}
 	}
 	Ctrl struct {
-		Endpoint              *UpdatableAddress
+		Endpoints             []*UpdatableAddress
```

Comment:
definitely would love to see the updated controllers get written to a state file. 👍